src/tests/dates.test.ts

```diff
@@ -442,3 +447,62 @@ describe('isYesterday', () => {
     expect(isYesterday(differentYear)).toBe(false);
   });
 });
+
+describe('isTomorrow', () => {
+  it('returns true for dates with same day, month, and year as the day after today', () => {
+    const today = new Date();
+    const tomorrow = new Date(today.getTime());
+    tomorrow.setDate(today.getDate() + 1);
+    expect(isTomorrow(tomorrow)).toBe(true);
+
+    // Time is irrelevant
+    const differentMinutes = new Date(tomorrow.getTime());
+    differentMinutes.setMinutes(tomorrow.getMinutes() + 1);
+    expect(isTomorrow(differentMinutes)).toBe(true);
+
+    const differentHours = new Date(tomorrow.getTime());
+    differentHours.setHours(tomorrow.getHours() + 1);
+    expect(isTomorrow(differentHours)).toBe(true);
+  });
+
+  it('returns false for dates with different day, month, or year from today', () => {
+    const today = new Date();
+    const tomorrow = new Date(today.getTime());
+    tomorrow.setDate(today.getDate() + 1);
+
+    const differentDay = new Date(tomorrow.getTime());
+    differentDay.setDate(tomorrow.getDate() + 1);
+    expect(isTomorrow(differentDay)).toBe(false);
+
+    const differentMonth = new Date(tomorrow.getTime());
+    differentMonth.setMonth(tomorrow.getMonth() + 1);
+    expect(isTomorrow(differentMonth)).toBe(false);
+
+    const differentYear = new Date(tomorrow.getTime());
+    differentYear.setFullYear(tomorrow.getFullYear() + 1);
+    expect(isTomorrow(differentYear)).toBe(false);
+  });
+});
+
+describe('isLeapYear', () => {
+  it('returns false for year divisible by 4 and 100 but not divisible by 400', () => {
```

Comment:
Would be nice to see a test of the opposite of this (that it's true for a year divisible by 4 and 100, like 2000)